pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = { file = ["requirements.txt"] }
 dev = { file = ["requirements-dev.txt"] }
 extra = { file = ["requirements-extra.txt"] }
 doc = { file = ["requirements-doc.txt"] }
+no-deps = { file = ["requirements-no-deps.txt"] }
```

Comment:
Why do we need to split between `dev` and `no-deps`? I think, we can pin `graphix-qasm-parser` in `requirements-dev.txt` while we need to install a specific branch.